spec/Functions.tex

```diff
@@ -722,17 +728,8 @@ \subsection{The Ref Return Intent}
 The \chpl{ref} return intent is specified by following the argument list with
 the \chpl{ref} keyword.  The function must return or yield an lvalue.
 
-During function resolution, if a procedure with a \chpl{ref} return
-intent and a procedure without a \chpl{ref} return intent are both
-possible candidates for a call, and either both use promotion or neither do,
-then this pair of procedures is called a ref-pair.
-
-When the call to a ref-pair is on the left-hand side of an
-assignment statement or appears in a call and corresponds to a formal argument
-with \chpl{out}, \chpl{inout}, or \chpl{ref} intent, the \chpl{ref} return
-intent procedure is used. Otherwise, the other candidate is used.  Thus,
-a ref-pair can be useful for adjusting the behavior depending on the
-calling context.
+It is a program error to return a variable with the \chpl{ref} return
+intent if that variable is used after it is destroyed.
```

Comment:
Hi Mike - I agree with everything in that paragraph. It sounds like the consequence in your mind is that it doesn't belong in the spec. Therefore, I will remove this sentence and the next appearance of it (in the const ref section).